spec/models/file_depot_nfs_spec.rb

```diff
@@ -12,12 +12,8 @@
       file_depot_nfs.uri = uri
     end
 
-    it "should return the uri set on the depot object and ignore the uri parameter" do
-      expect(file_depot_nfs.merged_uri(swift_uri, nil)).to eq uri
-    end
-
-    it "should return the uri set on the depot object and ignore an empty uri parameter" do
-      expect(file_depot_nfs.merged_uri(nil, nil)).to eq uri
+    it "should ignore the uri set on the depot object and return the uri parameter" do
+      expect(file_depot_nfs.merged_uri(swift_uri, nil)).to eq swift_uri
```

Comment:
Hrm... should `swift_uri` be the variable we use here when this is the `nfs` spec?

Reply:
Correct.  I cleaned this up, labeling the two URIs more intuitively.